src/components/VideoLayout/VideoLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import styles from './VideoLayout.module.scss';
+
+type Props = {
+  inlineLayout: boolean;
+  videoDetails: React.ReactNode;
+  videoDetailsInline: React.ReactNode;
+  inlinePlayer: React.ReactNode;
+  cinemaPlayer: React.ReactNode;
+  relatedVideosGrid: React.ReactNode;
+  relatedVideosList: React.ReactNode;
+};
+
+const VideoLayout: React.FC<Props> = ({
+  inlineLayout,
+  videoDetails,
+  videoDetailsInline,
+  inlinePlayer,
+  cinemaPlayer,
+  relatedVideosList,
+  relatedVideosGrid,
```

Comment:
Does it make sense to have separate props for the variants or would it make more sense to use:

```
{
  videoDetails,
  player,
  relatedVideos,
  inline,
}
```

The only downside is that you'll have to do ternary operators where you use the VideoLayout e.g.

```
<VideoLayout
  inline={isInline}
  videoDetails={isInline ? <InlineVideoDetails ...props /> : <VideoDetails ...props />}
  ...
/>
```

@RCVZ @dbudzins

Reply:
I would suggest moving the components into VideoLayout and passing necessary props through instead.

Reply:
Yes I do think is increases legibility